docs/content/dagster-cloud/account/managing-users.mdx

```diff
@@ -1,92 +1,128 @@
-
-- **Users are managed on a per-deployment basis**. **Organization Admins** are the exception and have access to the entire organization.
-
-  For example, if you have two full deployments (`prod` and `dev`), users who aren't **Organization Admins** must be added to each deployment to have access.
-
-- **If using Google for SSO**, users must be added in Dagster Cloud before they can log in.
-
-- **If using a SAML-based solution like Okta**, users must be assigned to the Dagster app in the SSO portal to log in. By default, users will be granted Viewer permissions on each deployment. The default role can be adjusted by modifying the [`sso_default_role` deployment setting](/dagster-cloud/developing-testing/deployment-settings-reference#sso-default-role).
-
-To add a new user to a deployment:
-
-1. Sign in to your Dagster Cloud account.
+<TODO></TODO>
 
-2. Click the **user menu (your icon) > Cloud Settings**.
-
-3. Fill in the following:
-
-   - **Email** - Enter the user's email address
-   - **Role** - Select the [user role](#understanding-user-permissions) for the user. **Note**: With the exception of the **Organization Admin** role, this role will only apply to the full deployment you're adding the user to.
-
-   For example:
-
-   <Image
-   alt="Cloud Settings Interface for Permissions"
-   src="/images/dagster-cloud/user-token-management/cloud_settings_permissions.png"
-   width={842}
-   height={346}
-   />
+---
 
-4. Click **+ Add**.
+## Understanding user roles
 
-### Removing a user
+<TabGroup>
+  <TabItem name="Overview">
 
-To remove a user from a deployment:
+### User roles
 
-1. Sign in to your Dagster Cloud account.
-2. Click the **user menu (your icon) > Cloud Settings**.
-3. Locate the user in the user list.
-4. Click **Remove**.
-5. When prompted, confirm the removal.
+Dagster Cloud supports six user roles for several levels of role-based access control:
 
-**Note**: This won't remove users from other deployments. For example, if a user has been added to both `prod` and `dev` but only removed in `prod`, they'll still be a user in `dev`.
-
----
-
-## Understanding user permissions
+<table>
+  <thead>
+    <tr>
+      <th width="20%; fixed">Role</th>
+      <th width="15%; fixed">Plan</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <strong>No access</strong>
+      </td>
+      <td>All plans</td>
+      <td>No access to the specified deployment/code location.</td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Viewer</strong>
+      </td>
+      <td>Enterprise</td>
+      <td>
+        The least permissive role. Grants access to view definitions and runs.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Launcher</strong>
+      </td>
+      <td>Enterprise</td>
+      <td>
+        Includes all Viewer permissions and the ability to launch and terminate
+        runs.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Editor</strong>
```

Comment:
Launcher and Editor are the two that can be overridden on Code Locations, dunno if we want to call that out in this list

Reply:
It gets called out later in the specific code locations section, so I think it's ok to leave it out here. We haven't introduced the concept of overrides yet, so it could be confusing.